README.rst

```diff
@@ -976,6 +980,15 @@ to new devices.
 This feature works also when editing group templates or the group assigned
 to a device via the `REST API <#change-device-group-detail>`__.
 
+Group Configuration Variables
+#############################
+
+Groups allow to define configuration variables which are automatically
+added to the device's context in the **System Defined Variables**.
+
+This feature works also when editing group templates or the group assigned
+to a device via the `REST API <#change-device-group-detail>`__.
```

Comment:
Let's add a brief section about meta data which contains the information being moved from above to here and clarify the difference between variables (context) and metadata. Make sure the API endpoint is linked.